xdsl/dialects/stablehlo.py

```diff
@@ -0,0 +1,41 @@
+from xdsl.dialects.builtin import TensorType
+from xdsl.ir import Attribute, Dialect, SSAValue
+from xdsl.irdl import IRDLOperation, irdl_op_definition, operand_def, result_def
+from xdsl.utils.exceptions import VerifyException
+
+
+@irdl_op_definition
+class AbsOp(IRDLOperation):
```

Comment:
Could you please add a docstring to this class, and to the top of this file? Ideally with links to the documentation for the dialect/op, and the overall purpose of the dialect.